delivery_carrier_label_batch/zpl_utils.py

```diff
@@ -3,6 +3,77 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
 
 
+def assemble_zpl2_single_images(files):
+    """Assemble a list of ZPL2 files with single image definition."""
+    zpl_images = dict()
+    zpl_labels = dict()
+    # Extract all the images and labels from different files
+    for file_nr, zpl_file in enumerate(files):
+        # ^XA stands for label start
+        images_def, label_def = zpl_file.split("^XA")
```

Comment:
It must be so, as "^XA" defines the start of a label: https://www.zebra.com/content/dam/zebra/manuals/printers/common/programming/zpl-zbi2-pm-en.pdf (page 347)

However, looking into this I'm realizing that graphics (or images) could be defined in many different ways (cf page 158 in PDF above), and this code is based on the zpl labels we get from the postlogistics webservice...

So looking at this it seems I should rework the feature to target graphics through "DGx" instead of "~DGR:IMG" :sob: 
Or we could add a hook in `delivery_carrier_label_batch` and move this into the postlogistics module, I'm not sure what's best, but I don't think we can manage all the cases defined by zpl specs anyway...

Reply:
Let's start with the case we know, can be adapted if another need comes later.